controller/api/Deliveries.php

```diff
@@ -220,4 +211,33 @@ public function publish(ServerRequest $request)
             $this->returnFailure($e);
         }
     }
+
+    private function getDeliveryParameter(?array $requestData): string
```

Comment:
either $requestData should be an array or you have to check that this is array

Reply:
Correct, fixed